src/components/Process/View.tsx

```diff
@@ -71,6 +75,39 @@ export const ProcessView = () => {
     if (election?.status === ElectionStatus.RESULTS) setTabIndex(1)
   }, [election])
 
+  useEffect(() => {
+    if (!err) return
+
+    if (err === 1) return setErr((prev) => prev + 1)
+
+    const htmlCollection = electionRef?.current?.children[0].children[0].children
+
+    if (htmlCollection) {
+      const array = Array.from(htmlCollection)
+
+      const elementsWithInvalidData = array.filter((el: any) => {
+        const dataInvalid =
+          el.children[0]?.children[1]?.children[1]?.children[0]?.children[0]?.getAttribute('data-invalid')
+        return dataInvalid === ''
+      })
+
+      if (elementsWithInvalidData.length) {
+        const firstElementWithInvalidData = elementsWithInvalidData[0].children[0].children[1]
+          .children[0] as HTMLElement
+        firstElementWithInvalidData.setAttribute('tabIndex', '0')
+        firstElementWithInvalidData.focus()
+      }
+    }
+  }, [err])
```

Comment:
Considering the amount of accesses to arbitrary array index positions, all this `useEffect` block should be inside a try/catch just in case.

Edit: We gotta talk about how you solved this because I have my doubts about all this DOM traversing.